src/server.ts

```diff
@@ -269,7 +269,7 @@ export class InversifyExpressServer {
                     value();
                 } else if (!res.headersSent) {
                     if (value === undefined) {
-                        res.status(204);
+                        res.status(200);
```

Comment:
do not we have http statuses object in this library ?

Reply:
No we have not.